wallet/invoices.c

```diff
@@ -70,6 +70,8 @@ static bool wallet_stmt2invoice(sqlite3_stmt *stmt, struct invoice *inv)
 		inv->paid_timestamp = sqlite3_column_int64(stmt, 9);
 	}
 
+	inv->bolt11 = tal_strndup(inv, sqlite3_column_blob(stmt, 11), sqlite3_column_bytes(stmt, 11));
```

Comment:
Why index 11? Does not match what you did in `invoices_load`, where you inserted `bolt11` between `expiry_time` and `pay_index`, which means you need to adjust every index from `pay_index` onwards.  Finally, this change should be done in the same commit as the change in `invoice_load`, since this function and the `SELECT` in `invoices_load` need to be in sync.